indico/modules/events/papers/templates/_contributions.html

```diff
@@ -31,6 +36,56 @@ <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
     {% endif %}
 {% endmacro %}
 
+{% macro render_editables_section(contribution) %}
+    {# TODO: Support mulitple editable rendering #}
+    {% set editable = contribution.editables[0] if contribution.editables[0] else none %}
```

Comment:
did what i suggested in my earlier comment not work?

Reply:
I thought that since were both temporary approaches and we will eventually iterate over all editables to not change it for now.